### ISSUE TITLE
讨论一下模型训练，tensorboard展示loss曲线的问题，各个loss之间的超参数该如何设置问题Discuss the problem of model training, Tensorboard displaying loss curves, and how to set hyperparameters between each loss

### ISSUE THREAD

**Issue by love112358** [yl4579/StarGANv2-VC#91]
大家好，这是我用自己数据训练网络模型，使用tensorboard工具绘制出不同loss函数的曲线图，根据不同的曲线图的走势，我该如何进一步调整各个loss之间的超参数，让不同的loss值表现的更好。你们调节不同loss 之间的超参数的根据是什么，大家能讨论分享一下吗？
Hello everyone, this is how I trained the network model with my own data and used the Tensorboard tool to draw curves of different loss functions. Based on the trend of different curves, how can I further adjust the hyperparameters between each loss to make different loss values perform better. Can you discuss and share the basis for adjusting hyperparameters between different losses?
![image](https://github.com/yl4579/StarGANv2-VC/assets/33959491/fea4d3c4-2a37-43c1-a3bb-ec930ef9ba9f)

**Comment by yl4579** [comment]
It looks weird that your F0 and ASR losses are constantly increasing and the cycle loss stays roughly the same after epoch 30. I think it's most likely because of not enough data and it looks like overfitting to me. How many hours of training data are you using in total?
